python/lsst/pipe/tasks/ingest.py

```diff
@@ -78,7 +78,10 @@ def getInfo(self, filename):
                 break
             ext = self.getExtensionName(md)
             if ext in extnames:
-                infoList.append(self.getInfoFromMetadata(md, info=phuInfo.copy()))
+                hduInfo = self.getInfoFromMetadata(md, info=phuInfo.copy())
+                # We need the HDU number when registering MEF files.
+                hduInfo["hdu"] = extnum - 1
```

Comment:
This makes `hdu` a very special dataId key name.  Have you checked if this could affect other obs packages?  
I wonder if we want this to be added to some higher-level documentations.